addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -42,7 +42,7 @@ const TableOfContent = publicWidget.Widget.extend({
         const $mainNavBar = $('#oe_main_menu_navbar');
         position += $mainNavBar.length ? $mainNavBar.outerHeight() : 0;
         position += isHorizontalNavbar ? this.$target.outerHeight() : 0;
-        $().getScrollingElement().scrollspy({target: '.s_table_of_content_navbar', method: 'offset', offset: position + 100});
+        $().getScrollingElement().scrollspy({target: '.s_table_of_content_navbar', method: 'offset', offset: window.innerHeight - 30});
```

Comment:
Seems strange, what does this change do? You also keep lines of code which change the position variable's value but you don't use position anymore...

Reply:
The idea was to set `windows.innerHeight - 30` as offset for "scrollspy", so each element that reaches this offset from screen bottom (when scrolling) will be selected, but I think this is not a good idea since it doesn't handle the case where we have a long table of content at the top of screen (the last element above offset will be always selected, which is a strange behaviour).

I think we can simply set the first element as 'active' on `Start`, this way, a visible table of content will have an "active" element by default and the selection will be correctly updated on scroll.